packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx

```diff
@@ -164,6 +164,7 @@ export class VizRepeater<V, D = {}> extends PureComponent<Props<V, D>, State<V>>
 
     const repeaterStyle: React.CSSProperties = {
       display: 'flex',
+      padding: '4px',
```

Comment:
this causes the panel to be misaligned 😬 

https://user-images.githubusercontent.com/20999846/205041745-f2fc6c0b-f871-4585-a441-e88bc463bb8f.mp4

Reply:
Yep. Not entirely sure about that. I could remove the overflow: hidden from the parent as it might break the panel to scroll

Reply:
@lpskdl let's remove this padding and deliver without it... it's still an improvement (you can at least tab to the bar now! 🥳), and we can investigate showing the focus ring in a different ticket. wdyt? 🤔

Reply:
sounds good :) Thanks Ash